etc/dnf/plugins/post-transaction-actions.conf

```diff
@@ -0,0 +1,3 @@
+[main]
+enabled = 1
+actiondir = /etc/dnf/plugins/post-transaction-actions.d/
```

Comment:
A question, why are we shipping a config file with values that are the default anyway?

Reply:
Consistency? Explicit is better then implicit? All other plugins do it this way.

Reply:
Hmm.. okay :slightly_smiling_face: